### ISSUE TITLE
can you add the comment section?

### ISSUE THREAD

**Issue by gute1** [Edsol/clickup-vscode-extension#8]
so we can see previous comments and add new comment

**Comment by Edsol** [comment]
yes, I can implement it. i will do it as soon as possible, thanks for the suggestion

**Comment by gute1** [comment]
just brought you a coffee for the last version.
I will be happy if you can add the comments for the next version. 
I promise another cup of coffee :)

**Comment by Edsol** [comment]
refactoring was needed before starting this implementation, now it will be much easier. Thank you so much for the support, I really appreciate it

**Comment by Edsol** [comment]
I am working on it, it will be available soon

**Comment by Edsol** [comment]
feature introduced in the new version
